routing/src/rib/vrf.rs

```diff
@@ -163,6 +163,10 @@ impl RouterVrfConfig {
         self.vni = Some(vni);
         self
     }
+    pub fn set_vni_opt(mut self, vni: Option<Vni>) -> Self {
```

Comment:
Why not just call this `set_vni`?  We have plenty of other similarly named methods that take an Option

Reply:
Yes. There are currently 3 methods to set the vni. I should probably unify them all.

Reply:
Ok, do you want to do that here, or later?